main/handlersettingscommon.go

```diff
@@ -140,3 +145,23 @@ func unmarshalProtectedSettings(configFolder string, hs handlerSettingsCommon, v
 	}
 	return nil
 }
+
+// cleanUpSettings clears out the settings file [ex: 0.settings] to ensure no
+// protected settings are logged in VM
+func cleanUpSettings(configFolder string) error {
```

Comment:
Would you be able to this common function ? - TryClearExtensionScriptsDirectoriesAndSettingsFilesExceptMostRecent()

https://github.com/Azure/azure-extension-platform/pull/30/files